src/main/resources/db/changelog/changes/vine-changelog-2023.04.12.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+        http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd">
+    <changeSet id="Add sold_wine to vine" author="volodymyr">
+        <addColumn tableName="vine">
+            <column name="sold_wine" type="int(5)" defaultValueComputed="0">
```

Comment:
why did u use here 'defaultValueComputed' property?

Reply:
I don't want to provide this field with null value in case no wine sold,  so decided to put 0

Reply:
Why don't u use defaultValueNumeric?

Reply:
I don't know)
Will update